great_expectations/expectations/expectation.py

```diff
@@ -1239,8 +1253,9 @@ def validate_configuration(
         except AssertionError as e:
             raise InvalidExpectationConfigurationError(str(e))
 
+    # Renamed from validate due to collision with Pydantic method of the same name
     @public_api
-    def validate(  # noqa: PLR0913
+    def validate_(  # noqa: PLR0913
```

Comment:
Happily taking suggestions to improve this name - I think we're going to chip away at the Expectation API when I'm done so I'm not sure we'll even need this method down the line.